vm/systemSmartContracts/validator.go

```diff
@@ -2121,7 +2098,7 @@ func (v *validatorSC) changeOwnerAndRewardAddressOnStaking(registrationData *Val
 	return vmcommon.Ok
 }
 
-//nolint
+// nolint
```

Comment:
can we check here as well the suppressed warning?

Reply:
unused method, removed it